tests/bson/bson-encode-002.phpt

```diff
@@ -36,38 +36,38 @@ class NumericArray implements BSON\Serializable, BSON\Unserializable
 }
 
 echo "Testing top-level AssociativeArray:\n";
-$bson = BSON\fromArray(new AssociativeArray);
-echo BSON\toJSON($bson), "\n";
+$bson = fromPHP(new AssociativeArray);
+echo toJSON($bson), "\n";
 echo "Encoded BSON:\n";
 hex_dump($bson);
-$value = BSON\toArray($bson, array("document" => 'AssociativeArray'));
+$value = toPHP($bson, array("root" => 'AssociativeArray'));
```

Comment:
Why was the document -> root change here needed?
AFAICT this test has now been repurposed to cover both "document" and "root" ?

I'd prefer to leave it covering "document" and adding another one for "root"